### ISSUE TITLE
Internal error - A non-numeric value encountered

### ISSUE THREAD

**Issue by Halama** [keboola/db-writer-mssql#29]
```
A non-numeric value encountered {"data":[],"errFile":"/code/vendor/keboola/db-writer-common/src/Application.php","errLine":94} []
```
Introduced in release 4.0.0. It looks like PHP message caused by switch to strict types.

**Comment by Halama** [comment]
Without stack trace #28 it is almost impossible to find the root cause.

But I think it is thrown here https://github.com/keboola/db-writer-mssql/blob/446691453f9a5c464269d3c09494abbac3b85d98/src/Writer/BCP.php#L118

Because of this 
![image](https://user-images.githubusercontent.com/903531/39914003-02ac9b22-5504-11e8-8264-8e41a63fa984.png)

It didn't throw error on older PHP version.

**Comment by Halama** [comment]
Previous version https://connection.keboola.com/admin/projects/2703/jobs/391076052
Current version https://connection.keboola.com/admin/projects/2703/jobs/391076082

**Comment by Halama** [comment]
FIXED IN #27
